doc/sphinx/docs/cpp/tutorials/cpp_tut.rst

```diff
@@ -0,0 +1,22 @@
+.. _cpp_tutorials:
+
+C++ tutorials
+================
+
+Basics
+^^^^^^
+
+After following the :ref:`install` you will be able to compile and run C++ pagmo
```

Comment:
I think here we should use the same format of the python tutorials, where after a generic introduction on how to use the tutorials (which for the c++ case could include the compilation line for gcc), we give a list of tutorials, each on a separate page.

This would ensure that we can build and add tutorials in the future without getting a long one pager.

Reply:
Sure. But right now there is only this example. Do you want to move this example to its own file? I'm not sure if I'll have time to add additional tutorials, but maybe this could be the start point for others to start writing them.

Reply:
Indeed thats the idea .... other user can then make PRs adding tutorials easily